.svglintrc.js

```diff
@@ -473,6 +473,43 @@ module.exports = {
                 ignoreIcon(reporter.name, iconPath, $);
               }
             }
+          },
+          function(reporter, $, ast) {
+            reporter.name = "path-format";
+
+            const iconPath = $.find("path").attr("d");
+            if (!updateIgnoreFile && isIgnored(reporter.name, iconPath)) {
+              return;
+            }
+
+            const validPathFormatRegex = /^[Mm][MmZzLlHhVvCcSsQqTtAaEe0-9-,.\s]+$/;
+            if (iconPath.match(validPathFormatRegex) === null) {
+              let errorMsg = "Invalid path format: "
+
+              if ("Mm".indexOf(iconPath[0]) === -1) {
```

Comment:
How about using a regular expression here as well?

```suggestion
              if (/^[Mm]/.test(iconPath) === -1) {
```

Reply:
I'm not against it, but will be computationally more expensive, and is something that in the long term worries me